testing/test_mark.py

```diff
@@ -1127,3 +1127,35 @@ def test_foo():
     result = pytester.runpytest(foo, "-m", expr)
     result.stderr.fnmatch_lines([expected])
     assert result.ret == ExitCode.USAGE_ERROR
+
+
+@pytest.mark.parametrize(
+    "markers",
+    (
+        "mark1",
+        "mark1 and mark2",
+        "mark1 and mark2 and mark3",
+        "mark1 and mark2 and mark3 and mark4",
+    ),
+)
+def test_markers_from_multiple_inheritances(pytester: Pytester, markers) -> None:
+    py_file = pytester.makepyfile(
+        """
+        import pytest
+        @pytest.mark.mark1
+        class Base1:
+            pass
+
+        @pytest.mark.mark2
+        class Base2:
+            pass
+
+        @pytest.mark.mark3
+        class TestMultipleInheritances(Base1, Base2):
+            @pytest.mark.mark4
+            def test_multiple_inheritances(self):
```

Comment:
@RonnyPfannschmidt 
What is the way to get the test markers?

Reply:
do you mean `node.iter_markers' ?

own_markers are those on just a specific node, iter_markers walks all of them

Reply:
I want to verify the `test_multiple_inheritances` contains the following markers:
* mark1
* mark2
* mark3
* mark4

The way I use is not correct...

Reply:
i see, take the `request` fixture

roughly 
```python

names= {mark.name for mark in node.iter_markers()}
assert names == {"mark1","mark2","mark3","mark4",}
```

there also is a api to collect specific nodes to help with that, bu that can be a followup